procedures/unit-testing-assertion-checks.ipf

```diff
@@ -198,10 +198,15 @@ static Function AreWavesEqual(wv1, wv2, mode, tol, detailedMsg)
 	variable mode, tol
 	string &detailedMsg
 
-	variable result
+	variable result, err
 
 	// handle NaN return values from EqualWaves for unknown modes
-	result = EqualWaves(wv1, wv2, mode, tol) == 1
+	try
+		result = EqualWaves(wv1, wv2, mode, tol) == 1; AbortOnRTE
```

Comment:
as note: this also gives an RT now when tol is NaN, Inf or -Inf.

Reply:
Which we are okay with I guess.